vcd/config_test.go

```diff
@@ -82,9 +82,10 @@ type TestConfig struct {
 		Org         string `json:"org"`
 		Vdc         string `json:"vdc"`
 		ProviderVdc struct {
-			Name           string `json:"name"`
-			NetworkPool    string `json:"networkPool"`
-			StorageProfile string `json:"storageProfile"`
+			Name            string `json:"name"`
+			NetworkPool     string `json:"networkPool"`
+			StorageProfile1 string `json:"storageProfile1"`
```

Comment:
At least the `sample_govcd_test_config.yaml` needs update. Could we possibly leave names `StorageProfile` and `StorageProfile2` for backwards compatibility of these configs?

Reply:
sample_govcd_test_config.yaml is already updated. I could leave `StorageProfile` but wanted to more explicitly readable with StorageProfile1 and StorageProfile2. @dataclouder what is your choice?

Reply:
Are they going to be both mandatory? What happens when we have a server with only one?

Reply:
in the configuration file we have already a `StorageProfile2` under `testEnvBuild`. I would like to avoid confusion

Reply:
In my test I made mandatory if one is missing, test will be skipped. Previous test working the same.

Reply:
> in the configuration file we have already a `StorageProfile2` under `testEnvBuild`. I would like to avoid confusion

so what is proposal?

Reply:
Removing the field from `testEnvBuild` and update the code that uses it to get `StorageProfile2` instead

Reply:
changed that

Reply:
after some missteps in my tests, I have to agree with Dainius. The field `StorageProfile1` should go back to `StorageProfile`

Reply:
renamed back